services/121-service/seed-data/fsp/fsp-commercial-bank-ethiopia.json

```diff
@@ -0,0 +1,25 @@
+{
+  "fsp": "Commercial-bank-ethiopia",
+  "integrationType": "api",
+  "fspDisplayNamePaApp": {
+    "en": "Commercial Bank of Ethiopia"
+  },
+  "fspDisplayNamePortal": "Commercial Bank of Ethiopia",
+  "questions": [
+    {
+      "name": "bankAccountNumber",
+      "label": {
+        "en": "Which bank account do you want to receive your money transfer from Commercial Bank of Ethiopia on?"
+      },
+      "placeholder": {
+        "en": "1000263499216"
```

Comment:
@Vladimir-Petkov this seems a quite specific placeholder.

Reply:
Added more generic phone number.